### PR TITLE
Add icon/logo for a new brand AeonMatrix and for its new integration Yardian

### PR DESCRIPTION
**Pull request by h3l1o5**
## Type of change

- [x] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [ ] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: https://github.com/home-assistant/core/pull/97326
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/28391
- Link to integration documentation on our website: https://www.home-assistant.io/integrations/yardian

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @h3l1o5 👋 

This PR isn't linked to a PR in our core repository, yet, this is adding a new brand and integration to the core repository?

Am I missing something? Or did you forget to link things up?

../Frenck

**Comment by h3l1o5** [comment]
Hi @frenck 

I haven't made a PR to the core repo yet. Thought I could make things done here first 😅.
I'll come back editing this PR as soon as the PR to the core repo is ready.
Thanks for the information 😄

File: core_integrations/yardian [hunk 1]
```diff
@@ -0,0 +1 @@
+../core_brands/aeonmatrix
```
Comment (frenck) [inline core_integrations/yardian:1]: This should not be using brands, as the parent PR in the core registry also isn't using brands. Please move the images into the `core_brands/aeonmatrix` folder directly.

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

File: core_integrations/yardian (hunk 1 above)
Comment (h3l1o5) [reply core_integrations/yardian]: Hey @frenck 
Got it! I assume that you meant move all the images into `core_integration/yardian` and remove `core_brands/aeonmatrix` right?

**Review by frenck — APPROVED** [review]
Thanks, @h3l1o5 👍 

../Frenck

**Pull request merged**